lib/ansible/modules/cloud/docker/docker_compose.py

```diff
@@ -717,6 +717,21 @@ def cmd_up(self):
             result['changed'] = build_output['changed']
             result['actions'] += build_output['actions']
 
+        if self.remove_orphans:
+            containers = self.client.containers(
+                filters={'label': ['{0}={1}'.format(LABEL_PROJECT, self.project.name), '{0}={1}'.format(LABEL_ONE_OFF, "False")]}
+            )
+
+            def _findOrphans():
```

Comment:
I would rewrite this (both there and in the other PR) as follows:
```.py
    for container in containers:
        service_name = container.get('Labels', {}).get(LABEL_SERVICE)
        if service_name not in self.project.service_names:
            result['changed'] = True
            return
```
You don't need the complete list of orphans, you only need to know whether there is at least one.

Reply:
While this is strictly true, I can see that it would be useful to provide the list of orphaned services that were removed.

Reply:
Getting rid of that local function and putting it directly into the list is still somewhat easier to read IMO :)